kolibri/plugins/user/assets/src/actions.js

```diff
@@ -10,7 +10,7 @@ function redirectToHome() {
 }
 
 function showRoot(store) {
-  const userSignedIn = coreGetters.isUserLoggedIn(store.state);
+  const userSignedIn = coreGetters.isUserSignedIn(store.state);
```

Comment:
note - if we change the core getters it needs to be updated in the external plugin now. I'm going to actually delete the files in the local plugin for now to prevent confusion. please install the VF plugin and commit these updates to there